crates/bevy_gizmos/src/circles.rs

```diff
@@ -4,15 +4,15 @@
 //! and assorted support items.
 
 use crate::prelude::Gizmos;
-use bevy_math::{Quat, Vec2, Vec3};
+use bevy_math::{Angle, Quat, Radians, Vec2, Vec3};
 use bevy_render::color::Color;
 use std::f32::consts::TAU;
 
 pub(crate) const DEFAULT_CIRCLE_SEGMENTS: usize = 32;
 
 fn circle_inner(radius: f32, segments: usize) -> impl Iterator<Item = Vec2> {
     (0..segments + 1).map(move |i| {
-        let angle = i as f32 * TAU / segments as f32;
+        let angle = Radians(i as f32 * TAU / segments as f32);
```

Comment:
The purpose of this was to make it clearer that the variable uses radians, I don't see why we couldn't have the explicitness in internal code too. But I guess it is quite clear since `sin_cos` is used right below, so I can remove this